build.sbt

```diff
@@ -407,23 +416,27 @@ lazy val codegenPlugin = (projectMatrix in file("modules/codegen-plugin"))
  */
 lazy val protocol = projectMatrix
   .in(file("modules/protocol"))
-  .jvmPlatform(buildtimejvmScala2Versions, jvmDimSettings)
+  .jvmPlatform(
+    autoScalaLibrary = false,
+    scalaVersions = Seq.empty,
+    settings = jvmDimSettings
+  )
+  .settings(
+    libraryDependencies += Dependencies.Smithy.model,
+    javacOptions ++= Seq("--release", "8")
```

Comment:
:+1:

Reply:
at least, in the JVM 11, `javac -Xlint` did not seem to be a thing